src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1530,10 +1530,47 @@ def receive_timeseries_from_csv(
         compute_timeseries_properties(dict_asset)
 
 
+def replace_nans_in_timeseries_with_0(timeseries, label):
+    """
+
+    Replaces nans in the timeseries (if any) with 0
+
+    Parameters
+
+    ----------
+    timeseries: pd.Series
+        timeseries in dict_asset (having nan value(s) if any), also of parameters
+
+    label: str
+        Key to print into the error message
```

Comment:
```suggestion
        Contains user-defined information about the timeseries to be printed into the eventual error message
```

One could (should) maybe use the `name` attribute of the `pd.Series` object instead of having a separate `label` no (see comment above)?

Reply:
I think we do not really use that attribute in the MVS right now. I generally agree, but then I would like to be consistend and change it for all `pd.Series` that we define.